CMakeLists.txt

```diff
@@ -43,24 +45,18 @@ else()
     endmacro()
 endif()
 
-# Use system zlib on unix and Qt ZLIB on Windows
 find_package(ZLIB REQUIRED)
 
-# All build libraries are moved to this directory
-set(LIBRARY_OUTPUT_PATH ${CMAKE_BINARY_DIR})
-
-set(LIB_SUFFIX "" CACHE STRING "Define suffix of directory name (32/64)")
-set(LIB_DESTINATION "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}" CACHE STRING "Library directory name" FORCE)
-set(INSTALL_PKGCONFIG_DIR "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/pkgconfig" CACHE STRING "Installation directory for pkgconfig (.pc) files" FORCE)
-set(QUAZIP_LIB_TARGET_NAME ${QUAZIP_LIB_VERSION_PREFIX}Quazip CACHE
+set(QUAZIP_LIB_TARGET_NAME quazip${QUAZIP_LIB_VERSION_SUFFIX} CACHE
```

Comment:
This reverts the lib name to the old scheme, see commit c32b9d4c3d164440e9f65c206e8f60d0e427bf48

Reply:
Sorry, overlooked that change.